src/main/java/taxi/controller/driver/LogoutController.java

```diff
@@ -0,0 +1,8 @@
+package taxi.controller.driver;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+
+@WebServlet(urlPatterns = "/drivers/logout")
+public class LogoutController extends HttpServlet {
+}
```

Comment:
Is logout controller works without any logic inside?

Reply:
Created with login and forgot, cause in task it is not needed).